integrations/facebook-pixel/lib/index.js

```diff
@@ -626,6 +626,10 @@ FacebookPixel.prototype.formatTraits = function formatTraits(analytics) {
   if (this.options.keyForExternalId) {
     external_id = traits[this.options.keyForExternalId]; // eslint-disable-line
   }
+  if (!external_id && analytics) { // eslint-disable-line
```

Comment:
Would be possible to enable this using a new option? something like `this.options.userIdAsExternalId`? when we create the documentation for that setting, I'll mention that we fallback to anonymousId if the userId is not present. Facebook Pixel is one of our most used destinations and I would like to give the option to the customers to toggle this new feature :) Thanks!